docs/resources/samples/add-excel-comments.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Add comments in Excel'
+description: 'Learn how to use Office Scripts to add comments in a worksheet.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# Add comments in Excel
+
+This sample shows how to add comments to a cell with at-mention of a contact.
```

Comment:
"At-mention" feels like an inside term.
```suggestion
This sample shows how to add comments to a cell that mention a colleague.
```

Reply:
@AlexJerabek In support docs, the term "@mention" is used though. Example: [Use @mentions to get someone's attention](https://support.microsoft.com/en-us/office/use-mentions-to-get-someone-s-attention-90701709-5dc1-41c7-aa48-b01d4a46e8c7)

What about...?
```suggestion
This sample shows how to add comments to a cell including @mentioning a colleague.
```

Reply:
Good call. I like your suggested change.